devel/revive-hf.lua

```diff
@@ -0,0 +1,40 @@
+-- Revives the specified historical figure
+
+--[====[
+
+devel/revive-hf
+=============
+
+Revivs the specified historical figure, even if off-site.
+
+Usage::
+
+    devel/revive-hf HISTFIG_ID
```

Comment:
`full-heal` is already associated with healing and reviving units. Instead of adding a new script, could we perhaps add a `--hf` parameter to `full-heal` so it can work on historical figures?

Reply:
I'm not sure. You wouldn't want to add --hf as a parameter to full-heal with this particular code though. 

Full-heal will only target units that are within the current map. And you can't mess with a unit's body data until they're loaded onto the map, as far as I know.

A work around that Rumrusher suggested, that would line up with full-heal's current functionality, is to spawn in a creature matching a dead histfig's description and then to link the unit's nemesis/soul together up with a new creature.